src/browser/HostInstaller.cpp

```diff
@@ -52,7 +52,13 @@ HostInstaller::HostInstaller()
 {
 }
 
-bool HostInstaller::checkIfInstalled(SupportedBrowsers browser)
+/**
+ * Checks if the selected browser has native messaging host properly installed
+ *
+ * @param browser Selected browser
+ * @return bool Script is installed correctly
+ */
+bool HostInstaller::checkIfInstalled(SupportedBrowsers browser) const
```

Comment:
When you changed this to a const function you fail to build on windows because "registryEntryFound" is not a const function.